terraform/development/main.tf

```diff
@@ -0,0 +1,28 @@
+# INSTRUCTIONS: 
+# 1) ENSURE YOU POPULATE THE LOCALS 
+# 2) ENSURE YOU REPLACE ALL INPUT PARAMETERS, THAT CURRENTLY STATE 'ENTER VALUE', WITH VALID VALUES 
+# 3) YOUR CODE WOULD NOT COMPILE IF STEP NUMBER 2 IS NOT PERFORMED!
+# 4) ENSURE YOU CREATE A BUCKET FOR YOUR STATE FILE AND YOU ADD THE NAME BELOW - MAINTAINING THE STATE OF THE INFRASTRUCTURE YOU CREATE IS ESSENTIAL - FOR APIS, THE BUCKETS ALREADY EXIST
+# 5) THE VALUES OF THE COMMON COMPONENTS THAT YOU WILL NEED ARE PROVIDED IN THE COMMENTS
+# 6) IF ADDITIONAL RESOURCES ARE REQUIRED BY YOUR API, ADD THEM TO THIS FILE
+# 7) ENSURE THIS FILE IS PLACED WITHIN A 'terraform' FOLDER LOCATED AT THE ROOT PROJECT DIRECTORY
+
+provider "aws" {
+  region  = "eu-west-2"
+  version = "~> 2.0"
+}
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+locals {
+  application_name = "resident information api" # The name to use for your application
```

Comment:
Would we use the kebab-case naming convention (like [here](https://github.com/LBHackney-IT/mosaic-resident-information-api/blob/0d4e5f50bc5c123d6832de5994c653ecaefe4667/terraform/development/main.tf#L40)) for this name also?

Appreciate that's a slightly random example from a VPC definition so may not be relevant here.

Not sure where this local is used, actually - will have a look around the files now!

Reply:
We don't seem to be setting it in the other repos so I wasn't sure. Happy to change it as it's probably safer to have it like that. (also never heard the term kebab-case before 😂 )

Reply:
*goes off and Googles kebab-case* (new one to me, thank you!)

Reply:
Searched for it and found these: camelCase and PascalCase i'd heard of...
But snake_case and kebab-case - not so much!